seqr/views/utils/orm_to_json_utils.py

```diff
@@ -757,10 +757,30 @@ def get_json_for_matchmaker_submission(submission):
         additional_model_fields=['contact_name', 'contact_href', 'submission_id'])
 
 
-def get_json_for_rna_seq_outliers(models, **kwargs):
-    additional_values = {
-        'isSignificant': Case(
-            When(p_adjust__lt=models.model.SIGNIFICANCE_THRESHOLD, then=Value(True)), default=Value(False)
-        ),
-    }
-    return get_json_for_queryset(models, additional_values=additional_values, **kwargs)
+def get_json_for_rna_seq_outliers(filters, significant_only=True):
+    filters['sample__is_active'] = True
+    data_by_individual_gene = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
```

Comment:
One note about this data structure is that if there are individuals that have one data type and not the other, it means that instead of having a n empty dictionary for the other data type the key will be missing entirely. 
Previous implementation:
```
{'outliers': {<data>}, 'spliceOutliers': {}}
```
Now:
```
{'outliers': {<data>}
```
I don't think this is a big deal, but we just need to make sure the UI can handle that and we should make sure its unit tested. If you did want to always have both keys you could change your definition here as follows, but it is also okay the way it is
```
data_by_individual_gene = defaultdict(lambda: {'outliers': defaultdict(list), 'spliceOutliers': defaultdict(list)})
```